scripts/migrate-to-page-bundles.py

```
@@ -0,0 +1,249 @@
+    return removed
+
+
+def main():
+    print(f"Hugo Page Bundle Migration")
+    print(f"  Content dir: {CONTENT_DIR}")
+    print(f"  Static dir:  {STATIC_DIR}")
+    print()
+
+    # Collect all .md files
+    md_files = []
+    for dirpath, dirnames, filenames in os.walk(CONTENT_DIR):
```

Ruff (0.15.2)

[warning] Loop control variable `dirnames` not used within loop body

Rename unused `dirnames` to `_dirnames`

(B007)